__tests__/app.test.js

```diff
@@ -285,6 +285,23 @@ describe('DELETE /api/comments/:comment_id', () => {
         })
     })
 })
+describe('GET /api/users', () => {
+    it('returns status 200 and array of user objects with the follwoing keys: username, name, avatar_url', () => {
+        return request(app)
+        .get('/api/users')
+        .expect(200)
+        .then(({body}) => {
+            const userExample = {
+                username: expect.any(String),
+                name: expect.any(String),
+                avatar_url: expect.any(String)
+            }
+            body.users.map((user) => {
```

Comment:
is there a particular reason you're using a map instead of a forEach. a map returns a new array which you aren't using or really need.